libraries/AP_HAL_Linux/RCOutput_AioPRU.cpp

```diff
@@ -20,7 +20,11 @@
 
 #include <AP_HAL/AP_HAL.h>
 
-#include "../../Tools/Linux_HAL_Essentials/pru/aiopru/RcAioPRU_bin.h"
+#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBLUE
+#include "../../Tools/Linux_HAL_Essentials/pru/aiopru/RcAioPRU_BBBLUE_bin.h"
+#else
```

Comment:
@magicrub Thanks for the review Tom.

`#else` is right here, I think, because when you compile a none BBBmini or none BeagleBone Blue Linux board, with `#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI`, `PRUcode` is not defined and the compile will stop with an error. Now `RcAioPRU_BBBMINI_bin.h` is included and compiling is fine.